build.py

```diff
@@ -1509,8 +1509,9 @@ def dockerfile_prepare_container_linux(argmap, backends, enable_gpu, target_mach
 """
 
     if target_platform() not in ["igpu", "windows", "rhel"]:
-        repo_arch = "sbsa" if target_machine == "aarch64" else "x86_64"
-        df += f"""
+        if FLAGS.triton_wheels_dependencies_group != "cpu" :
+           repo_arch = "sbsa" if target_machine == "aarch64" else "x86_64"
+           df += f"""
```

Comment:
The reason behind this installation is that PyTorch backend is looking for that library, and PyTorch is part of the CPU build process. 

In the same time, using flag `--triton-wheels-dependencies-group` is not applicable here that flag is designed for different purpose, it's for wheel dependency group options.